gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -123,7 +123,7 @@ ext.externalDependency = [
     "guiceMultibindings": "com.google.inject.extensions:guice-multibindings:4.0",
     "guiceServlet": "com.google.inject.extensions:guice-servlet:4.0",
     "derby": "org.apache.derby:derby:10.12.1.1",
-    "mockito": "org.mockito:mockito-inline:4.11.0", // upgraded to allow mocking for constructors, static and final methods; specifically for iceberg distcp
+    "mockito": "org.mockito:mockito-core:4.11.0",
```

Comment:
Add a comment here as a warning not to change this to mockito-inline. If you need features from mockito-inline, please add mockito-inline as a mockmaker plugin

 Example: https://github.com/apache/gobblin/pull/3651/files#r1163182255

Reply:
https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable

Reply:
@homatthew yup makes sense... I will add a comment to avoid updating the dependency to mockito-inline